lammps-tutorials.tex

```diff
@@ -103,7 +105,9 @@
 \maketitle
 
 \begin{abstract}
-The availability of open-source molecular simulation codes allows scientists and engineers to run simulations without parameterizing interparticle interactions or writing code for time-integration schemes. Although performing molecular simulations is now accessible to a large audience, the wide array of options and possible pitfalls when using these open-source tools can complicate their use, particularly for beginners. LAMMPS is a versatile molecular simulation code designed for modeling particle-based systems for a broad range of materials science and computational chemistry applications. The software package is widely used to run simulations on large-scale parallel computing environments, making it ideal for complex and extensive simulations of atomic and molecular systems. This article introduces a suite of tutorials designed to make learning LAMMPS more accessible to new users. The first four tutorials cover the basics of running molecular simulations in LAMMPS with systems of varying complexities, including a simple fluid and a carbon nanotube. The last three tutorials address more advanced molecular simulation techniques, specifically the use of a reactive force field, enhanced sampling, and grand canonical Monte Carlo.
+  The availability of open-source molecular simulation software packages allows scientists and engineers to focus on running and analyzing simulations without having to write, parallelize, and validate their own simulation software.  While molecular simulations thus become accessible to a larger audience, the ``black box'' nature of such software packages and wide array of options and features as well as possible pitfalls can make it challenging to use them correctly, particularly for beginners in the topic of simulations.  LAMMPS is one such versatile molecular simulation code designed for modeling particle-based systems for a broad range of materials science and computational chemistry applications from atomistic, over coarse--grained, mesoscale, and grid-free continuum models to discrete element modeling.  LAMMPS is capable of efficiently running simulations of varying sizes from small desktop computer to large-scale supercomputing environments. Its flexibility and extensibility makes it ideal for complex and extensive simulations of atomic and molecular systems and beyond.  This article introduces a suite of tutorials designed to make learning LAMMPS more accessible to new users.  The first four tutorials cover the basics of running molecular simulations in LAMMPS with systems of varying complexities, including a simple fluid and a carbon nanotube.  The last three tutorials address more advanced molecular simulation techniques, specifically the use of a reactive force field, enhanced sampling, and grand canonical Monte Carlo.
```

Comment:
suggest rephrasing to ...." chemistry applications, including atomistic, coarse--grained, mesoscale, and grid-free continuum models."

Reply:
Thanks, done.